### PR TITLE
Fix #357: Reload preview when files are deleted and renamed

### LINKED ISSUE #357: Update blob urls on file delete
STR:
1. Add an image into the fs
2. Add an image tag referencing the image in an html file
3. Delete the image from the fs

The image is still shown/rendered in the preview pane. We should revoke the blob url (maybe refresh the preview too) so that the image disappears from the preview.

### PR DESCRIPTION
**Pull request by humphd**
This was harder to fix than I thought, because the obvious ways of fixing it caused infinite reload loops.  I've added code to guard against re-entrant reloads for the same URL.

### CODE REVIEW

File: src/extensions/default/bramble/lib/launcher.js [hunk 1]
```diff
@@ -35,6 +35,10 @@ define(function (require, exports, module) {
                 return;
             }
             browser.update(urlOrHTML);
+
+            if(typeof callback === "function") {
```
Comment (sedge) [inline src/extensions/default/bramble/lib/launcher.js]: Why do you need to check the type here?

Comment (humphd) [inline src/extensions/default/bramble/lib/launcher.js]: I'm breaking the contract on `launch`, which should be sync and have no callback at all.  Callers from within Brackets code proper won't use it.  I only use it for my own needs elsewhere in our code.

Comment (sedge) [inline src/extensions/default/bramble/lib/launcher.js]: Okay, makes sense.

**Pull request merged**